hack/verify-govet.sh

```diff
@@ -1,27 +1,26 @@
 #!/bin/bash
 
+set -o errexit
```

Comment:
Fixed, now the Travis output should include the message "FAILURE: go vet failed!", instead of "make: *** [verify] Error 1" ([old job output](https://travis-ci.org/openshift/source-to-image/jobs/130451051#L274)).